Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/mobs_hostile_mercenaries.yml

```
@@ -113,3 +114,49 @@
     - MobMercenarySpecialistMedic
     - MobMercenarySpecialistAssault
     rareChance: 0.2
+
+#region space
+- type: entity
+  id: SpawnMobMercenaryEVAT1
+  parent: SpawnMobMercenaryT1
+  components:
+  - type: RandomSpawner
+    prototypes:
+    - MobMercenarySoldierEVAKnife
+    - MobMercenarySoldierEVAPistol
+    - MobMercenarySoldierEVANovalite
+    chance: 1
+    offset: 0.0
+    rarePrototypes:
+    - SpawnMobMercenaryEVAT2
+    rareChance: 0.15
```

```suggestion
    rareChance: 0.05
```
Is this intentionally different from the rest at 5%?